app/models/person.rb

```diff
@@ -218,7 +218,7 @@ def self.username_blacklist
   end
 
   def self.username_available?(username, community_id)
-    !username.in?(USERNAME_BLACKLIST) &&
+    !username.downcase.in?(USERNAME_BLACKLIST) &&
```

Comment:
although adding to_set seemed to break a validation where that list was used, so left that out for now, but did the other change to use include? rather than in?